packages/itwinui-css/src/surface/surface.scss

```diff
@@ -18,9 +18,37 @@
   border-radius: var(--iui-surface-border-radius);
   box-shadow: var(--iui-surface-elevation);
   color: var(--iui-color-text);
+
+  &:where([data-iui-layout='true']) {
+    display: flex;
+    flex-direction: column;
+  }
+}
+
+$surface-padding: var(--iui-size-2xs);
+
+@mixin iui-surface-header {
+  display: flex;
+  flex-shrink: 0;
+  align-items: center;
+  overflow: hidden;
```

Comment:
does this need overflow hidden?

Reply:
It looks like removing `overflow: hidden` nullifies `scrollbar-gutter: stable both-edges`.  We also risk text leaking out of the container.
![Screenshot 2023-02-23 at 11 22 38 AM](https://user-images.githubusercontent.com/849817/220967613-9392c8fb-f854-45c1-a08b-b9a3ab265add.png)

Reply:
ok, lets keep it then